src/components/AnalyticsPieChart.tsx

```diff
@@ -39,7 +39,7 @@ const AnalyticsPieChart = () => {
           datasets: [
             {
               data: top5Visits,
-              label: "# of Votes",
+              label: "No. of Views",
```

Comment:
No. of views feels kinda redundant.
Would it improve clarity if we just write `Views: 5` or `Viewed by 5` or `Viewed 5 times`.
Personally I like  `Views: 5`